src/code/z_room.c

```diff
@@ -232,10 +275,10 @@ s32 func_80096238(void* data) {
             osSyncPrintf("成功…だと思う。 time = %6.3f ms \n", OS_CYCLES_TO_USEC(time) / 1000.0f);
             // "Writing back to original address from work buffer."
             osSyncPrintf("ワークバッファから元のアドレスに書き戻します。\n");
-            // "If the original buffer size isn't at least 150kb, it will be out of control."
+            // "If the original buffer size isn't at least 150kB, it will be out of control."
             osSyncPrintf("元のバッファのサイズが150キロバイト無いと暴走するでしょう。\n");
 
-            bcopy(gZBuffer, data, sizeof(gZBuffer));
+            bcopy(gZBuffer, data, sizeof(u16[SCREEN_HEIGHT][SCREEN_WIDTH]));
```

Comment:
the data is a screen height*width rgba16 image before it is the same size as the depth framebuffer, their size being the same (ofc gZBuffer needs to be at least as large) and in particular the 2d array type and dimensions being the same is "just" a detail/coincidence

Reply:
Yeah I suppose you can see it as being the size of `data` instead `gZBuffer`, since that's what it copies to. It's still a bit unfortunate to hardcode the type here but w/e

Reply:
I wish we could `static_assert(sizeof(gZBuffer) >= ...)` yeah